modules/acm/variables.tf

```
@@ -62,6 +62,12 @@ variable "create_ssh_key" {
   default     = true
 }
 
+variable "ssh_auth_key" {
+  description = "Key for Git authentication. Overrides 'create_ssh_key' variable."
+  type        = string
+  default     = ""
```

We should use `null` for empty defaults when possible.

```suggestion
  default     = null
```

---

Set to `""` because it's used in ["local-exec" command](https://github.com/paulpalamarchuk/terraform-google-kubernetes-engine/blob/add_var_to_pass_ssh-key_in_acm/modules/acm/main.tf#L75)